extending/javascript-client/index.md

```diff
@@ -0,0 +1,183 @@
+---
+title: JavaScript Client
+---
+The REST API includes a JavaScript/Backbone client library.
+
+The library provides an interface for the WP REST API by providing Backbone Models and Collections for all endpoints exposed the API Schema.
+
+## Using
+
+Activate the WP-API plugin. Enqueue the script directly:
+
+```php
+wp_enqueue_script( 'wp-api' );
+```
+
+or as a dependency for your script:
+
+```php
+wp_enqueue_script( 'my_script', 'path_to_my_script', array( 'wp-api' ) );
+```
+
+The library parses the root endpoint (the 'Schema') and creates matching Backbone models and collections. You will now have two root objects available to you: `wp.api.models` and `wp-api.collections`.
```

Comment:
`wp.api.models` but `wp_api.collections`? Underscore looks like a typo, either way it's inconsistent